templates/_base.html.twig

```diff
@@ -1,6 +1,191 @@
-{% include 'header.twig' %}
-<div id="body">
-    {% block body %}
-    {% endblock %}
+<!DOCTYPE html>
+<html lang="{{ lang }}" prefix="og: http://ogp.me/ns#">
+<head>
+    <title>{{ title }} | Björn Büttner</title>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" href="/styles.css"/>
+    <meta name="author" content="{{ author.name }}"/>
+    <meta name="description" content="{{ description }}"/>
+    <meta name="og:title" content="{{ title }}"/>
+    <meta name="og:description" content="{{ description }}"/>
+    <meta name="og:image" content="https://bjoern-buettner.me/logo.png"/>
+    <meta name="og:url" content="https://bjoern-buettner.me{{ (active ~ '/' ~ lang)|fixurl }}"/>
+    <meta name="og:site_name" content="Björn Büttner"/>
+    <meta name="og:locale" content="{{ lang }}"/>
+    <meta name="og:type" content="{{ og_type }}"/>
+    <link rel="canonical" href="https://bjoern-buettner.me{{ (active ~ '/' ~ lang)|fixurl }}"/>
+    <link rel="alternate" hreflang="x-default" href="https://bjoern-buettner.me{{ (active ~ '/en')|fixurl }}"/>
+    <link rel="alternate" hreflang="en" href="https://bjoern-buettner.me{{ (active ~ '/en')|fixurl }}"/>
+    <link rel="alternate" hreflang="de" href="https://bjoern-buettner.me{{ (active ~ '/de')|fixurl }}"/>
+    <link rel="alternate" type="application/rss+xml" href="https://bjoern-buettner.me/rss-{{ lang }}.xml"/>
+</head>
+<body>
+<div class="header">
+    <a href="/{{ lang }}"><img src="/logo.png" alt="Logo"/></a>
+    <h1>Bj&ouml;rn B&uuml;ttner</h1>
```

Comment:
I think we don't need a h1 here - this is not the topic of the page. Please use a strong instead.